### PR TITLE
Sources: add by Location view

### PR DESCRIPTION
**Pull request by lukarenko**

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -681,15 +681,17 @@
             }
         },
         "sources": {
-            "title": "Sources of Infection and Quarantine",
+            "title": "Sources of Infection and Location",
             "titleMenu": "Sources of Infection",
-            "disclaimerGeneral": "Note: data for last week are not complete yet - share of imported cases may be smaller than actual.",
+            "disclaimerGeneral": "Note: data for last week are not complete yet - number of cases maybe be lower than actual.",
```
Comment (drola) [inline src/locales/en.json:686]: may be

File: src/locales/en.json [hunk 2]
```diff
@@ -681,15 +681,17 @@
             }
         },
         "sources": {
-            "title": "Sources of Infection and Quarantine",
+            "title": "Sources of Infection and Location",
```
Comment (drola) [inline src/locales/en.json:684]: Sources and Location of Infection

File: src/locales/sl.json [hunk 3]
```diff
@@ -1157,15 +1157,17 @@
             }
         },
         "sources": {
-            "title": "Viri okužb in karantena",
+            "title": "Viri okužb in lokacija",
```
Comment (drola) [inline src/locales/sl.json:1160]: Viri in lokacija okužb

**Review by drola — APPROVED** [review]
Code ok. A few comments on labels.

File: src/locales/en.json (hunk 1 above)
Comment (lukarenko) [reply src/locales/en.json:686]: tnx

File: src/locales/sl.json (hunk 3 above)
Comment (lukarenko) [reply src/locales/sl.json]: tnx

**Pull request merged**